### ISSUE TITLE
Cannot open the config

### ISSUE THREAD

**Issue by night2078** [Tape-Dispenser/Immersive-Music-Mod#17]
Hey there,

i've recently discovered your awesome mod and i'm really enjoying the new songs.

However, unfortunately i can't seem to get the /cfg command to work. It's just telling me, that the command does not exist. I'm using version 1.0.2 on 1.20.1 fabric, launched and installed using the modrinth launcher (not sure if that is important).

Am i doing something wrong or has the command been changed and the mod description is just outdated?

**Comment by Tape-Dispenser** [comment]
could you please navigate to your `.minecraft/logs` folder and attach `latest.log`

**Comment by night2078** [comment]
Sure thing

[latest.log](https://github.com/Tape-Dispenser/Immersive-Music-Mod/files/14968381/latest.log)

**Comment by Tape-Dispenser** [comment]
try relaunching the mod and running the other commands (`/np`, `/timmstop`, `/skip`)

**Comment by night2078** [comment]
Nah, none of them works.

Is it possible that it's related to me using the modrinth launcher? I mean, technically the mod is working, i do hear music that i've never heard before in minecraft. So i assume the mod is correctly working, but the commands seem to be missing.

Could also be an incompatible mod, but i do have way over 300 mods installed, that's just too much work for my taste to pinpoint the exact mod by disabling every single one seperately and test it again.

EDIT: Would also be interesting to know, if i am the only one with that issue. Meaby it is indeed related to installing the mod with modrinth launcher.

**Comment by Tape-Dispenser** [comment]
could you upload the log file again? I was also thinking it may have something to do with mod compatibility, but i'm not sure. According to your last log TIMM is running all it's initialization code but nothing else, but if you're hearing the music then maybe something else is going on. could you also check to see if `.minecraft/config/timm` exists, and that there are two files inside the folder?

**Comment by Tape-Dispenser** [comment]
I doubt that installing with the modrinth launcher is causing any issues, although it may be a good idea to try manually deleting/reinstalling the mod

**Comment by night2078** [comment]
[latest.log](https://github.com/Tape-Dispenser/Immersive-Music-Mod/files/14968610/latest.log)

Here's the latest log again

And yes, there is indeed a folder named "timm" with two files in it:
biome_playlist.json
TIMM.config

**Comment by Tape-Dispenser** [comment]
could you go into TIMM.config and make sure that debug is true? like so: `"debug":["bool","true"]`

**Comment by night2078** [comment]
Yes, changed it. Back into game i guess and give you my latest log, correct?

**Comment by Tape-Dispenser** [comment]
yes please

**Comment by night2078** [comment]
[latest.log](https://github.com/Tape-Dispenser/Immersive-Music-Mod/files/14968618/latest.log)

**Comment by Tape-Dispenser** [comment]
weird, everything seems to be running fine, just the commands don't work. my only thought is maybe one of the other mods is preventing the fabric client command manager or something. I'm going to be adding compatibility with mod menu in the next update so hopefully that should allow you to open the config menu. as for the song control commands, i've been thinking of either a gui or keybind interface for that as well, as the commands were only really meant as a temporary interface anyways. i might do some investigation to see what mod is causing this issue later also

**Comment by night2078** [comment]
Yea, i mean, it's nothing to serious of an issue for me.

The mod is technically working, so i guess it's okay. I'll do some further testing, like disabling one or two mods that i think could lead to such things, or try to use a different version.

Like you said, meaby it's getting fixed with the mod menu integration. we'll see. Anyway, still thanks for trying to help.

**Comment by night2078** [comment]
Hey,

just wanted to give you a short update,

after you've added the mod menu support, it didn't change the command problem, but i can access the config menu now, using mod menu. I mean, you could technically see it as solved, since you can now access the config without the need of the command.

Thanks for the workaround.

**Comment by Tape-Dispenser** [comment]
yeah, ideally id still like to know what is causing the issue with commands just not working. are you running a specific modpack i could download to investigate the compatibility issue?

**Comment by night2078** [comment]
Nah, i've picked them all myself. Not sure how, but i believe i could publish it as a modpack using the modrinth launcher, since i use that to keep my mods easily updated.
